File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int l = nums.length;
        int[] num1 = new int[2]; 

        for (int i = 0; i < l; i++) {
            for (int j = i + 1; j < l; j++) {
                if (nums[i] + nums[j] == target) { 
                    num1[0] = i;
                    num1[1] = j;
                    return num1;
                }
            }
        }

        
        return new int[0];
    }
}

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int> mpp;  // num -> index
        for(int i = 0; i < nums.size(); i++) {
            int complement = target - nums[i];
            
            if(mpp.find(complement) != mpp.end()) {
                return {mpp[complement], i};
            }
            
            mpp[nums[i]] = i;  // store current number with index
        }
        return {}; // not found
    }
};